src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -136,68 +129,241 @@
     private static final double LOG_ODDS_THRESHOLD = 3.0;
 
     /** Metrics for evaluating the performance of whole genome sequencing experiments. */
-    public static class WgsMetrics extends MetricBase {
+    public static class WgsMetrics extends MergeableMetricBase {
+
+        @NoMergingIsDerived
```

Comment:
you are abusing the annotation here...I guess we should just rename the annotation "NoMerging" since in this case, it's not actually derived but rather manually merged.